client/my-sites/plugins/plugin-management-v2/update-plugin/style.scss

```
@@ -22,21 +22,12 @@
 }
 
 button.update-plugin__new-version {
-	text-decoration: underline;
-	font-weight: 600;
+	background-color: var(--studio-jetpack-green-50);
+	color: #fff;
```

```suggestion
	color: var(--studio-white);
```

Nitpick: We could use the respective variable for the white color.

---

Got it: 4500248e05836b4fb09bf8b93bc0e81dbdb92f7e